File: swagger/models/get_bookable_items_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse

class GetBookableItemsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		availabilities
		Contains information about the availabilities for appointment booking.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Availabilities': 'availabilities',
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'availabilities': 'Availabilities',
		}

		'PaginationResponse': ('pagination_response', PaginationResponse),
	availabilities = None

File: swagger/Models/GetBookableItemsResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetBookableItemsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse
	 */
	public $paginationResponse;
	/**
	 * Contains information about the availabilities for appointment booking.
	 * 
	 * @var Availability[]
	 */
	public $availabilities;
}
